packages/inter-protocol/test/smartWallet/test-psm-integration.js

```diff
@@ -81,13 +84,28 @@ test('null swap', async t => {
       want: { Out: anchor.makeEmpty() },
     },
   });
+
   await eventLoopIteration();
 
+  t.like(updates[0], {
+    updated: 'balance',
+  });
+
+  const statusUpdateHasKeys = (updateIndex, result, numWants, payouts) => {
```

Comment:
@turadg I reverted these asserts because they only worked when the test was run in isolation, and we ended up changing `current` not `updates`